setup.py

```diff
@@ -37,6 +37,7 @@
     "openai": ["openai>=1.0"],
     "schemas": ["jsonschema"],
     "server": ["fastapi", "uvicorn"],
+    "cli": ["colored"]
```

Comment:
Not a super widely-used package, so I am open to alternative solutions if there are any concerns.

Reply:
I've used `colorama` before, which claimed better Windows support if I remember right? 

@nopdive might have thoughts here

Reply:
\begin{mutter}If you refactor so that notebook output and console output are separate implementations of the same base class, this problem largely goes away\end{mutter}

Reply:
The problem still stands of choosing a way to map probabilities to colors, but yes at the very least the problem of **parsing** a particular rgba value goes away :)

Reply:
@Harsha-Nori, I'm not familiar with `colored`, and `colorama` in its current state: not fussed which package is used as long as its maintained and we can confirm it works for our target set of environments.